inc/composer/class-docker-compose-generator.php

```diff
@@ -131,17 +132,17 @@ protected function get_php_reusable() : array {
 				'ELASTICSEARCH_HOST' => 'elasticsearch',
 				'ELASTICSEARCH_PORT' => 9200,
 				'AWS_XRAY_DAEMON_HOST' => 'xray',
-				'S3_UPLOADS_ENDPOINT' => "https://{$this->tld}/",
+				'S3_UPLOADS_ENDPOINT' => $this->set_url_scheme( "http://s3-{$this->hostname}.localhost/" ),
```

Comment:
Might have some related comments on the other PR about this, but using just the TLD was required due to the way the SDK generates URLs based on the configured region etc... and the way the Mnio/S3 server handles them to route appropriately.

We probably need to keep it pointing to `$this->tld` rather than localhost. Don't suppose you remember why this was @rmccue ?

Reply:
See this comment before making any changes https://github.com/humanmade/altis-local-server/pull/466#discussion_r857812295

Reply:
Yes; I fixed this separately, see line I commented on in inc/namespace.php